fireant/slicer/queries.py

```diff
@@ -106,9 +106,14 @@ def query_data(self, database, table, joins=None,
         query = self._build_data_query(table, joins, metrics, dimensions, dfilters, mfilters, references, rollup)
 
         query_string = str(query)
-        logger.info("Executing query:\n----START----\n{query}\n-----END-----".format(query=query_string))
+        logger.debug('Executing query:\n----START----\n{query}\n-----END-----'.format(query=query_string))
+        start_time = time.time()
 
         dataframe = database.fetch_dataframe(query_string)
+        logger.debug('Query completed [duration: {duration} seconds]:\n----START----\n{query}\n-----END-----'.format(
```

Comment:
OK - I was logging it twice in case we wanted to record any queries that failed to execute so that we could debug them.